rdkafka-sys/src/bindings.rs

```diff
@@ -976,7 +976,7 @@ pub enum rd_kafka_queue_s { }
 pub type rd_kafka_queue_t = rd_kafka_queue_s;
 #[derive(Copy, Clone)]
 #[repr(i32)]
-#[derive(Debug)]
+#[derive(Debug,PartialEq)]
```

Comment:
I don't think there is any option to add it automatically via bindgen (command i used: `bindgen --builtins --convert-macros librdkafka-sys/librdkafka/src/rdkafka.h > outout.rs`).

We should be able to implement it directly, and we could add the implementation in src/types.rs. I've tried with the following snippet of code and the compiler doesn't complain:

```rust
impl PartialEq for RDKafkaRespErr {                                                                                                                                                                                                             
    fn eq(&self, other: &RDKafkaRespErr) -> bool {
        self == other
    }
}
```